webapp/pages/index.vue

```diff
@@ -86,6 +99,7 @@ export default {
         },
         // Transform the previous result with new data
         updateQuery: (previousResult, { fetchMoreResult }) => {
+          this.hasMore = fetchMoreResult.Post && fetchMoreResult.Post.length > 0
```

Comment:
```suggestion
          this.hasMore = fetchMoreResult.Post && fetchMoreResult.Post.length >= this.pageSize
```
Would give better user experience.